apps/numgen.py

```diff
@@ -129,17 +129,24 @@ def setDB(self):
         """Sets the database to store the number."""
         self.dbName = self.generatorFrame.dbBox.currentText()
         self.viewerFrame.statusLabel.setText("database updated")
+        self.broadcastRequested.emit(
+            "logbus", 
+            f"Database to store is set as {self.dbName}." if self.dbName
+            else "Database to store is not selected."
+        )
 
     @pyqtSlot()
     def generateNumber(self):
         """Generates and shows a random number when the button is clicked."""
         # generate a random number
         num = generate()
         self.viewerFrame.numberLabel.setText(f"generated number: {num}")
+        self.broadcastRequested.emit("logbus", f"Generate a random number: {num}.")
```

Comment:
How about "Generated a random number:" or "Generated number:"?

Reply:
I think "Generated number:" is better.